addons/viewrestriction/initSettings.sqf

```diff
@@ -0,0 +1,9 @@
+[
+    QGVAR(preserveView),
+    "CHECKBOX",
+    [LSTRING(SettingPreserveViewName), LSTRING(SettingPreserveViewDesc)],
+    "ACEX " + localize LSTRING(ModuleDisplayName),
```

Comment:
Use `format`.

Reply:
why? It's slower and not more readable.

Reply:
You're right, for 2 strings it's slower. But such a tiny bit that I think just using format is better overall.

Reply:
In case of changes in the future, it's easier to add another, without risking worse performance loss. But this is so minimal in this case no one cares.

Reply:
> Actually why not just have "ACEX" as part of string?

It's already used in editor module name.